### PR TITLE
Command completion

### PR DESCRIPTION
**Pull request by mkinney**
Got command completion working in both bash and zsh.

Not as much completion options as I would have hoped for.

### CODE REVIEW

**Comment by mkinney** [comment]
Hmm... I was doing completion from a user perspective not from a developer perspective.

For instance, after enabling either, you should be able to run "pyinfra <tab>" or "pyinfra -<tab>" to see available options.

File: scripts/pyinfra-complete.sh [hunk 1]
```diff
@@ -1,21 +1,27 @@
-_pyinfra_completion() {
-    local IFS=$'
-'
-    COMPREPLY=( $( env COMP_WORDS="${COMP_WORDS[*]}" \
-                   COMP_CWORD=$COMP_CWORD \
-                   _PYINFRA_COMPLETE=complete $1 ) )
-    return 0
-}
+#!/usr/bin/env bash
```
Comment (Fizzadar) [inline scripts/pyinfra-complete.sh]: What's the reasoning with changing this script? The old/current one is dynamic so will always be up to date whereas this hardcodes the options?

**Comment by Fizzadar** [comment]
Info on click's autocomplete: https://click.palletsprojects.com/en/7.x/bashcomplete/

File: scripts/pyinfra-complete.sh (hunk 1 above)
Comment (mkinney) [reply scripts/pyinfra-complete.sh]: It didn't work for me when I tried it earlier. I'll re-test.

Comment (mkinney) [reply scripts/pyinfra-complete.sh]: Ok. It works. Not sure what I did last time.

Comment (mkinney) [reply scripts/pyinfra-complete.sh]: Will close this PR and open a new one.

**Pull request closed without merging**